macs/nix/nix-darwin.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
What is this JSON stuff for?

Reply:
To track and pin the versions of the "channels" being deployed, to keep it from being a floating version.